data/i18n/en/en.toml

```diff
@@ -279,6 +279,9 @@ other = "Thanks for the feedback. If you have a specific, answerable question ab
 [layouts_docs_search_fetching]
 other = "Fetching results..."
 
+[legacy_repos_message]
+other = """Kubernetes has [new package repositories hosted at `pkgs.k8s.io`](/blog/2023/08/15/pkgs-k8s-io-introduction/) starting from August 2023. The legacy package repositories (`apt.kubernetes.io` and `yum.kubernetes.io`) have been deprecated and frozen starting from September 13, 2023. To install Kubernetes versions released **after** September 13, 2013, you need to use the community-owned package repositories. Please read the [deprecation and freezing announcement](/blog/2023/08/31/legacy-package-repository-deprecation/) for more details. The new Kubernetes package repositories provide downloads for Kubernetes versions starting with v1.24.0."""
```

Comment:
I believe you meant 2023 where you input 2013.
Not sure why we use three quotes here.

The following texts are scary, and more important than some other texts in the current message:

> The existing packages in the legacy repositories will be available for the foreseeable future. However, the Kubernetes project can't provide any guarantees on how long is that going to be. The deprecated legacy repositories, and their contents, might be removed at any time in the future and without a further notice period.

Reply:
In [TOML](https://toml.io/en/), three quotes are for multiline strings. These allow the Markdown to contain newline characters, if needed. Wrapping long lines can help readability (for contributors).

Reply:
Cool, then please wrap the long line.

Reply:
> I believe you meant 2023 where you input 2013.

Fixed, thanks for pointing this out!

> Cool, then please wrap the long line.

Done!

> The following texts are scary, and more important than some other texts in the current message:

I'd prefer leaving the message as it is. We want to recap everything and be as explicit as possible. If there are strong concerns about this, we can try to figure out a better message together.

Reply:
@xmudrii I made this suggestion based on previous discussions on the impact on users. The most scary thing is not that "yay, we got a new home", it is instead "sorry, we cannot promise you that the old repo will still be there next month.". It is fine to link to the blogs or other sources if the current text is already too long. The whole point of adding this alert is to let users be aware of the impacts to them. BTW, the current text is too long not because we have covered a lot of important points, it is because the text is verbose.